ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3448,6 +3448,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-platform-external
+- as: vsphere-upi-ccm-external-f14
```

Comment:
I did not noticed you had ever added `vsphere-upi-platform-external-p2-f14` job. 

Shall we replace `vsphere-upi-platform-external-p2-f14` with this new one?

Reply:
That one was added in 4.13, that one is `None`, which is consistent with https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_14/upi-on-vsphere/versioned-installer-external_platform, and the new one is `External`, which is consistent with https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_14/upi-on-vsphere/versioned-installer-external_ccm, so seems we should keep the both.

Reply:
Shall we take `PATCH_INFRA_MANIFEST` into consideration?

Reply:
I meant what is the difference when setting `PATCH_INFRA_MANIFEST` or not.

Reply:
`PATCH_INFRA_MANIFEST` is false by default https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-ref.yaml#L19, do you mean to explicitly set it to false?

Reply:
I am just curious what case should use `PATCH_INFRA_MANIFEST: true`, if you can confirm the current job is doing the right thing, I will let it go.

Reply:
The dev's code reuses the code of 4.13, seems we should set `PATCH_INFRA_MANIFEST` to `true` for 4.13, but set `PATCH_INFRA_MANIFEST` to `false` for 4.14 and `false` is the default value . Let me update this for 4.13.
https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh#L173

Reply:
One more suggestion, can we update the job name to `vsphere-upi-platform-external-f14`, and update the other one - `vsphere-upi-platform-external-f14` to `vsphere-upi-platform-none-f14` in both 4.13 and 4.14, does that make sense?

Reply:
Sure, make sense. I just trigger the 4.13 job, let's wait the 4.13 result then update.

Reply:
The 4.13 cluster installed as expected. But sorry, I have more thought about your suggestion, and I prefer to keep the original job name now, because the platform shows `External`, it just treat `External` as `None`, and the feature was introduced in 4.13, I think we are already familiar with the `External` platform. And it's consistent with flexy-install. WDYT? @jianlinliu 
```
sh-4.4$ /cli/oc get clusterversion
NAME      VERSION                              AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.13.0-0.nightly-2023-08-31-163330   True        False         32m     Cluster version is 4.13.0-0.nightly-2023-08-31-163330
sh-4.4$ /cli/oc get node          
NAME              STATUS   ROLES                  AGE   VERSION
compute-0         Ready    worker                 42m   v1.26.7+0ef5eae
compute-1         Ready    worker                 41m   v1.26.7+0ef5eae
compute-2         Ready    worker                 41m   v1.26.7+0ef5eae
control-plane-0   Ready    control-plane,master   53m   v1.26.7+0ef5eae
control-plane-1   Ready    control-plane,master   53m   v1.26.7+0ef5eae
control-plane-2   Ready    control-plane,master   53m   v1.26.7+0ef5eae
sh-4.4$ /cli/oc get infrastrucuture cluster -oyaml
error: the server doesn't have a resource type "infrastrucuture"
sh-4.4$ /cli/oc get infrastructure cluster -oyaml
apiVersion: config.openshift.io/v1
kind: Infrastructure
metadata:
  creationTimestamp: "2023-09-01T08:01:44Z"
  generation: 1
  name: cluster
  resourceVersion: "502"
  uid: 4b6626c2-c655-4ad8-b96b-ff36209ec65c
spec:
  cloudConfig:
    name: ""
  platformSpec:
    external:
      platformName: vSphere
    type: External
status:
  apiServerInternalURI: https://api-int.ci-op-pgsn27sp-d22fd.vmc-ci.devcluster.openshift.com:6443
  apiServerURL: https://api.ci-op-pgsn27sp-d22fd.vmc-ci.devcluster.openshift.com:6443
  controlPlaneTopology: HighlyAvailable
  etcdDiscoveryDomain: ""
  infrastructureName: ci-op-pgsn27sp-d22fd-tzk5w
  infrastructureTopology: HighlyAvailable
  platform: External
  platformStatus:
    external: {}
    type: External
sh-4.4$ /cli/oc describe machineconfig 01-worker-kubelet | grep cloud-provider
      --cloud-provider= \
sh-4.4$ 

```

Reply:
Per your explanation in https://github.com/openshift/release/pull/42901/files#r1312634851, actually I am thinking if flexy-template naming is proper, even the implemention code need to align with the dev's codes.  But because flexy-template is being deprecated, I would not  ask a update for flexy-template.

Another reason that I can see in https://github.com/openshift/release/pull/39923, dev is using `platform-external` and `platform-none` for the job names, so I think we should keep consistent with the same style.

Reply:
ok, updated.